robot_skills/src/robot_skills/base.py

```diff
@@ -193,26 +198,37 @@ def move(self, position_constraint_string, frame):
 
         return plan
 
-    def force_drive(self, vx, vy, vth, timeout):
+    def force_drive(self, vx, vy, vth, timeout, ax=float('inf'), ay=float('inf'), ath=float('inf')):
         """ Forces the robot to drive by sending a command velocity directly to the base controller. N.B.: all collision
         avoidance is bypassed.
 
         :param vx: forward velocity in m/s
         :param vy: sideways velocity in m/s
         :param vth: rotational velocity in rad/s
+        :param ax: forward acceleration in m/s^2
+        :param ay: sideways acceleration in m/s^2
+        :param ath: rotational acceleration in rad/s^2
         :param timeout: duration for this motion in seconds
         """
         # Cancel the local planner goal
         self.local_planner.cancelCurrentPlan()
 
         v = geometry_msgs.msg.Twist()        # Initialize velocity
         t_start = rospy.Time.now()
+        t_end = rospy.Time.now() + rospy.Duration.from_sec(timeout)
+
+        def _abs_max(value, abs_max_value):
+            if value == 0:
+                return 0
+            return (value / abs(value)) * min(abs_max_value, abs(value))
```

Comment:
`(value / abs(value))` determines the sign of `value` so maybe numpy.sign() is a better alternative so that there is no need for the `if`? Although numpy isn't in here yet. 

Moreover, is the abs_max_value really absolute? Depends on use, but seems to me like another `abs` function should be used.